config/core/configmaps/features.yaml

```diff
@@ -39,3 +39,8 @@ data:
   # ALPHA feature: The subscriber-strict flag force subscriptions to define a subscriber
   # For more details: https://github.com/knative/eventing/issues/5756
   strict-subscriber: "disabled"
+
+  # ALPHA feature: The new-trigger-filters flag allows you to use the new `filters` field
+  # in Trigger objects with its rich filtering capabilities.
+  # For more details: https://github.com/knative/eventing/issues/5204
+  new-trigger-filters: "disabled"
```

Comment:
should we name this differently

`cncf-Subscription-filters` ?

Reply:
The name was part of [the proposal](https://hackmd.io/@devguyio/H1Mg3my2d). I think it's pros and cons, one reason for this name is to emphasize this being "new filter language" for the users. Any strong opinion there?

Reply:
> emphasize this being "new filter language" for the users. 

:100:  that is a great / strong point. Let's go that route!

+1 on `new-trigger-filters` :smiley: